installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc

```diff
@@ -33,7 +33,7 @@ include::modules/installation-network-user-infra.adoc[leveloffset=+2]
 
 .Additional resources
 
-* xref:../install_config/installing-customizing.adoc#installation-special-config-chrony_installing-customizing[Configuring chrony time service]
```

Comment:
the new path included 'crony', is it a typo or on purpose?

Reply:
It's an existing typo in the module name: https://github.com/openshift/openshift-docs/blame/master/modules/installation-special-config-crony.adoc

The build fails with the previous xref. Please confirm that you want me to update the filename.

Reply:
(I am leaning towards not changing it because we'd lose the file history when we renamed it.)